templates/registration/login.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Login</h1>
+  {% if form.errors %}
+    <p style="color: red">Invalid data</p>
+  {% endif %}
+
+  <form action="{% url 'login' %}" method="post">
+    {% csrf_token %}
+    {{ form.as_p }}
+    <input type="submit" name="Submit">
+    <input type="hidden" name="next" value={{ next }}>
```

Comment:
The 'next' parameter is included as a hidden input field. Ensure that the 'next' variable is properly passed to the template context to redirect users to their intended page after login. Additionally, consider wrapping the value in quotes to avoid issues with URLs containing special characters.